### ISSUE TITLE
Redirect loop on frontend

### ISSUE THREAD

**Issue by ethitter** [markoheijnen/wordpress-mu-domain-mapping#4]
I haven't isolated exactly why, but I regularly end up in a redirect loop when accessing the front end of a site I'm logged in to. If I view the admin and then go to the front end from there, the loop usually doesn't happen.

I suspect it has something to do with the cookies needing a refresh, or invalid values in the domain_mapping_logins table.

Whatever it is, it's annoying. I'll try to debug a bit further in the coming days.

**Comment by markoheijnen** [comment]
Are you using the master branch? That one can be a bit unstable and still need to find time to see what is causing that

**Comment by ethitter** [comment]
Nope, this happens using the current release from Extend. I've been experiencing this for a while, I just haven't had time to debug.
